src/docs/release/breaking-changes/material-chip-button-semantics.md

```diff
@@ -0,0 +1,223 @@
+with a `Semantics` widget marked as `button: true`. **In this case, the inner
+and outer `button` semantics will conflict, resulting in the tappable area of
+the button shrinking down to the size of the label after this change is
+introduced.** This can be fixed by deleting that `Semantics` widget and
+replacing it with its child, or simply removing the `button: true` property if
+other semantic properties still need to be applied to the label widget of the
+Chip.
+
+The following snippets use
+[InputChip](https://api.flutter.dev/flutter/material/InputChip-class.html) as an
+example, but the same process applies to
+[ActionChip](https://api.flutter.dev/flutter/material/ActionChip-class.html),
+[ChoiceChip](https://api.flutter.dev/flutter/material/ChoiceChip-class.html),
+and [FilterChip](https://api.flutter.dev/flutter/material/FilterChip-class.html)
+as well.
+
+**Case 1: remove the `Semantics` widget.**
+
+Code before migration:
+
+<!-- skip -->
+```dart
+Widget myInputChip = InputChip(
+  onPressed: () {},
+  label: Semantics(
+    button: true,
+    child: Text('My Input Chip'),
+  ),
+);
+```
+
+Code after migration:
+
+<!-- skip -->
+```dart
+Widget myInputChip = InputChip(
+  onPressed: () {},
+  label: Text('My Input Chip'),
+);
+```
+
+**Case 2: remove `button:true` from the `Semantics` widget.**
+
+Code before migration:
+
+<!-- skip -->
+```dart
+Widget myInputChip = InputChip(
+  onPressed: () {},
+  label: Semantics(
+    button: true,
+    hint: 'Example Hint',
+    child: Text('My Input Chip'),
+  ),
+);
+```
+
+Code after migration:
+
+<!-- skip -->
+```dart
+Widget myInputChip = InputChip(
+  onPressed: () {},
+  label: Semantics(
+    hint: 'Example Hint',
+    child: Text('My Input Chip'),
+  ),
+);
+```
+
+## Timeline
+
+Landed in version: TBD<br> In stable release: not yet
+
+## References
+
+{% include master-api.md %}
+
+API documentation:
+* [`Semantics`][]
+* [`SemanticsProperties.button`][]
+* [`SemanticsProperties.enabled`][]
+* [`Chip`][]
+* [`ActionChip`][]
+* [`ChoiceChip`][]
+* [`FilterChip`][]
+* [`InputChip`][]
+
+Relevant issues:
+* [flutter/flutter#58010][]
+
+Relevant PRs:
+* [Tweaking Material Chip a11y semantics to match buttons][]
+* [Revert "Tweaking Material Chip a11y semantics to match buttons (#60141)"][]
+* [Re-land "Tweaking Material Chip a11y semantics to match buttons (#60141)"][]
+
+[`Semantics`]: {{site.api}}/flutter/widgets/Semantics-class.html
+[`SemanticsProperties.button`]:
+{{site.api}}/flutter/semantics/SemanticsProperties/button.html
+[`SemanticsProperties.enabled`]:
+{{site.api}}/flutter/semantics/SemanticsProperties/enabled.html [`Chip`]:
+{{site.api}}/flutter/material/Chip-class.html [`ActionChip`]:
+{{site.api}}/flutter/material/ActionChip-class.html [`ChoiceChip`]:
+{{site.api}}/flutter/material/ChoiceChip-class.html [`FilterChip`]:
+{{site.api}}/flutter/material/FilterChip-class.html [`InputChip`]:
+{{site.api}}/flutter/material/InputChip-class.html
```

Comment:
I think this may have gotten mixed up by newlines

Reply:
Good catch, fixed now.